docs/software/sciapps/vasp.md

```diff
@@ -1,5 +1,368 @@
+Hybrid functionals that mix the Hartree-Fock approach with density functional theory are implemented as well.
+Furthermore, Green's functions methods (GW quasiparticles, and ACFDT-RPA) and many-body perturbation theory (2nd-order Møller-Plesset) are available in VASP.
+
+In VASP, central quantities, like the one-electron orbitals, the electronic charge density, and the local potential are expressed in plane wave basis sets.
+The interactions between the electrons and ions are described using norm-conserving or ultrasoft pseudopotentials, or the projector-augmented-wave method.
+To determine the electronic groundstate, VASP makes use of efficient iterative matrix diagonalisation techniques, like the residual minimisation method with direct inversion of the iterative subspace (RMM-DIIS) or blocked Davidson algorithms.
+These are coupled to highly efficient Broyden and Pulay density mixing schemes to speed up the self-consistency cycle.
+
+
+!!! note "Licensing Terms and Conditions"
+    Users are kindly asked to obtain their own license. CSCS cannot provide free access to the code and needs to inform VASP Software GmbH with an updated list of users.
+    Therefore, access to precompiled VASP.6 executables and library files will be available only to users who have already purchased a VASP.6 license and upon request will become members of the CSCS unix group vasp6.
+    Please refer to the VASP web site for more information.
+
+
+## Alps - GH200
+
+### How to run
+A precompiled user environment containing VASP with MPI, OpenMP, OpenACC, HDF5 and Wannier90 support is available.
+Due to license restrictions, the VASP images are not directly accessible in the same way as other applications.
+
+For accessing VASP uenv images, please follow this guide: [`Accessing Restricted Software`][ref-uenv-restricted-software].
+
+To load the VASP user environment:
+```bash
+uenv start vasp/v6.5.0:v1 --view=vasp
+```
+The `vasp_std` , `vasp_ncl`  and `vasp_gam`  executables are now available for use.
+Loading the uenv can also be directly done inside of a SLURM script.
+
+```bash title="SLURM script for running VASP on a single node"
+#!/bin/bash -l
+
+#SBATCH --job-name=vasp
+#SBATCH --time=24:00:00
+#SBATCH --nodes=1
+#SBATCH --ntasks-per-node=4
+#SBATCH --cpus-per-task=16
+#SBATCH --gpus-per-task=1
+#SBATCH --uenv=<VASP_UENV>
+#SBATCH --view=vasp
+#SBATCH --account=<ACCOUNT>
+#SBATCH --partition=normal
+
+export OMP_NUM_THREADS=$SLURM_CPUS_PER_TASK
+
+srun vasp_std
+```
+
+!!! note
+    It's recommended to use the SLURM option `--gpus-per-task=1`, since VASP may fail to properly assign ranks to GPUs when running on more than one node.
+    This is not required when using the CUDA MPS wrapper for oversubscription of GPUs.
+
+!!! note
+    VASP relies on CUDA aware MPI, which requires `MPICH_GPU_SUPPORT_ENABLED=1` to be set when using Cray MPICH. On the HPC platform including `daint`, this is set by default and does not have to be included in SLURM scripts.
```

Comment:
```suggestion
    VASP relies on CUDA-aware MPI, which requires `MPICH_GPU_SUPPORT_ENABLED=1` to be set when using Cray MPICH. On the HPC platform including `daint`, this is set by default and does not have to be included in SLURM scripts.
```
I think it would be good to leave this note, but still add the environment variable to the Slurm script. We decided to do the same for CP2K.